backend/app/utils/env.py

```
@@ -0,0 +1,15 @@
+
+# getting IP 
+ip_address = os.getenv('IP_ADD')
+
+if not ip_address:
+    raise RuntimeError("Missing required environment variable: IP_ADD")
+
+# getting GPU Config 
+gpu_config = os.getenv('GPU')
+
+if not gpu_config:
+    raise RuntimeError("Missing required environment variable or Invalid: GPU")
```

Ruff (0.15.15)

[warning] Avoid specifying long messages outside the exception class

(TRY003)